### ISSUE TITLE
Unclear what went wrong (massive stack trace)

### ISSUE THREAD

**Issue by sourishkrout** [google-github-actions/get-gke-credentials#270]
### TL;DR

CI job stopped working due to what was likely an intermittent error (resolved itself after some time) on GCP/GKE's side. Unclear what happened since GH action just traced its own source code.

### Expected behavior

An error message that includes an indicator of what went wrong, even if it just surfaces the upstream error message.

At the minimum, it'd be great not to trace all of the source file. Turns out there wasn't a needle in this haystack.

### Observed behavior

Appears on error the GitHub action traces it's own source file? Fwiw, the "stack trace" contains no trace of an underlying error.

### Action YAML

```yaml
https://github.com/stateful/linkerd-website/actions/runs/4545065193/workflow
```

### Log output

```text
https://gist.github.com/sourishkrout/03222ab313752d3aa5ac59224ac348a6
```

**Comment by sethvargo** [comment]
Hi @sourishkrout - we do our best to catch errors and return human friendly ones. The _entire_ function is wrapped in a try-catch. Unfortunately this error was thrown _before_ it made it to our code execution (it's like GitHub Actions failed to parse the javascript). I don't really think there's anything we can do here, since it never executed our code.
